### PR TITLE
Edit cursor.prev() method docs in lexer

### PR DESCRIPTION
**Pull request by pierwill**
Fix missing punctuation

### CODE REVIEW

File: src/librustc_lexer/src/cursor.rs [hunk 1]
```diff
@@ -23,8 +23,8 @@ impl<'a> Cursor<'a> {
         }
     }
 
-    /// For debug assertions only
-    /// Returns the last eaten symbol (or '\0' in release builds).
+    /// Returns the last eaten symbol (or `\0` in release builds).
```
Comment (pierwill) [inline src/librustc_lexer/src/cursor.rs:26]: Should this be `'\0'`?

Comment (matthewjasper) [reply src/librustc_lexer/src/cursor.rs:26]: yes

Comment (pierwill) [reply src/librustc_lexer/src/cursor.rs]: Fixed—

**Pull request merged**